addon-test-support/@ember/test-helpers/dom/trigger-event.ts

```diff
@@ -57,7 +57,7 @@ registerHook('triggerEvent', 'start', (target: Target, eventType: string) => {
 export default function triggerEvent(
   target: Target,
   eventType: string,
-  options?: object
+  options?: Record<string, unknown>
```

Comment:
1. 💙 
2. In the future, we should see if we can make this more specific, but it shouldn't block this change. Making it more accurate will "just" be a bug fix.

Reply:
Yeah, I did a deep dive on what the types should be, and `options` is just passed around willy nilly to a bunch of functions that take `any`, so figuring out the actual type will be a Project™️. I figured `Record<string, unknown>` was at least a little better than `object` and more consistent with another `options` type you specified for a similar test helper.

Reply:
I could do something like `Event & Record<string, unknown>` bc I'm pretty sure it's mostly used for specifying details on the `Event` to be triggered, and I think that would at least give some hints about the types for things that exist on an `Event`.

Reply:
Going to open a separate PR w/ the deep dive since I couldn't help myself. 😂

Reply:
Hahaha I love it. Feels very https://xkcd.com/356/ 😂 "A Project™" sounds exactly right, though.